index.html

```diff
@@ -117,6 +117,35 @@ <h3>Multi-Post Stories</h3>
             <button type="button" class="project-button">See Project</button>
         </section> 
     </section>
+    <section class="form-container">
+        <h3>CONTACT US</h3>
+        <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </P>
+        <form action="https://formspree.io/f/xdoylybq" method="POST">
+            <label for="name"></label>
+            <input type="text" id="name" name="Full name" placeholder="Text Input" required>
```

Comment:
Please change the placeholder text as was asked in the previous review. It should be meaningful for the user so that the user can input the relevant information.